bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -2708,8 +2708,11 @@ components:
                   example: NMFTA
                 isPartialLoadAllowed:
                   type: boolean
+                  default: true
                   description: |
                     Indication whether the shipper agrees to load part of the shipment in case where not all of the cargo is delivered within cut-off.
+  
+                    **Note:** If this property is omitted it is concidered that 'Partial Load **is** allowed' by default
```

Comment:
typo in considered

Reply:
Well spotted - fixed 3 other places as well